submissions/api.py

```diff
@@ -991,6 +992,11 @@ def _get_or_create_student_item(student_item_dict):
             student_item_serializer = StudentItemSerializer(
                 data=student_item_dict)
             if not student_item_serializer.is_valid():
+                logger.error(
+                    "Invalid StudentItemSerializer: {}".format(
+                        unicode(student_item_serializer.errors)
+                    )
```

Comment:
Reviewers: While I logged the field_errors above, I didn't actually log the `student_item_dict`. Now we know from Splunk that the problem is an invalid course_id - but we don't know what the course_id value actually is. ☹️ 🙍‍♂️ 🙍  😦